pupil-spa/src/app/user.service.ts

```diff
@@ -1,24 +1,40 @@
 import { Injectable } from '@angular/core';
-import * as responseMock from './login.response.mock.json';
+import 'rxjs/add/operator/toPromise';
+import { Http } from '@angular/http';
 const auth_token = 'auth_token';
 
 @Injectable()
 export class UserService {
   private loggedIn = false;
+  private apiURL = 'http://localhost:3001'
```

Comment:
This apiURL should be picked up from the environment.  how do we configure this for test and prod?